internal/handler/common/ismaster.go

```diff
@@ -51,5 +52,16 @@ func IsMasterDocuments() []*types.Document {
 		"maxWireVersion", MaxWireVersion,
 		"readOnly", false,
 		"ok", float64(1),
-	))}
+	))
+
+	if name != "" {
+		if strings.HasPrefix(host, ":") {
+			host = "localhost" + host
```

Comment:
This is not the right approach, but I'm not sure what would be better. Maybe a cli flag to set the host?..

If the app that wants to connect to FerretDB is not running on the same host as FerretDB, this field should contain the address to connect. Another way is probably to parse `rs.initiate` command and set hosts according to the params of this command.

Reply:
Reading the docs here it seems like `rs.initiate` would be better.
Any reason why you're considering a CLI flag besides being simpler?

Reply:
No, cli flag being simpler is the only reason. I don't see a quick way to implement `rs.initiate` command handling.

Reply:
Well, another reason is that this works only for TCP listen addresses and does not work for TLS-only setups…